protocol-specs/eth2_spec_tests/tests/general/phase0/bls/aggregate_verify/small/aggregate_verify_infinity_pubkey/data.yaml

```diff
@@ -0,0 +1,10 @@
+input:
```

Comment:
we don't need any tests for aggregate_verify. Not sure if it's possible to exclude them.

Reply:
As we already have the specs, I don't see any harm of having that function in library and testing it. The interfaces we will expose from `lisk-cryptography` will be different and we will not expose all blst library interfaces.

Reply:
ok